data/pref.xml

```diff
@@ -121,6 +121,7 @@
       <option id="size_limit" type="int" default="0" />
       <option id="goto_modified" type="bool" default="true" />
       <option id="allow_nonlinear_history" type="bool" default="false" />
+      <option id="show_tooltip" type="bool" default="true" migrate="Options.show_tooltip" />
```

Comment:
I have removed the `migrate` attribute. The migrate attribute is used to transform old options used in `get/set_config_string` functions to the new `pref.xml` file format. As this is a new option, it doesn't need a `migrate` (the given `Options.show_tooltip` never existed).